docs/architecture.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Anatomy of a Next Generation Kubernetes Distribution
+weight: 204
+---
+
+### Architecture Overview
+
+With RKE2 we take lessons learned from developing and maintaining our lightweight [Kubernetes][io-kubernetes]
+distribution, [K3s][io-k3s], and apply them to build an enterprise-ready distribution with [K3s][io-k3s] ease-of-use.
+What this means is that RKE2 is, at its simplest, a single binary to be installed and configured on all nodes expected
+to participate in the [Kubernetes][io-kubernetes] cluster. Once started, RKE2 is then able to bootstrap and supervise
+role-appropriate agents per node while sourcing needed content from the network.
```

Comment:
Should we include any reference to airgap capabilities here?

Reply:
We will have a dedicated airgap section. No need to reference in this doc.